New Wiki Major Enhancements

hauni97 opened the pull request:
Major improvements in the mining guide.

h4x3rotab commented on content/en-us/docs/khala-mining/1-1-hardware-requirements.md:
```diff
@@ -21,9 +21,12 @@ A system can potentially mine Phala if it meets these general requirements:
 - [x] Have a supported version of Ubuntu (18.04, 20.04, 21.04) installed and booted from it.
    - See [Suported OS requirements](#supported-operating-systems). 
+- [x] You require a minimum amount of 5 PHA to mine
+   - Furthermore, you require 2000 PHA tokens to stake. 
```
Not exactly 2000. It's based on some calculation

hauni97 replied:
Ok thx for the info. A formula might be better to calculate the minimum PHA staking requirements.

hauni97 replied:
TODO: - Minimal stake formula: A public Google Sheet for estimation (Suggest to go through the full mining process first, because a lot of factors impact the score)

hauni97 replied:
> 

This has now been fixed in this commit: https://github.com/Phala-Network/phala-wiki-next/pull/35/commits/5aa069c47e1f838f667f13dbfc3f756b89bf0c33

A spreadsheet link will be added.

hauni97 reviewed (commented):
@h4x3rotab please add any future additions here to this new wiki. The 'Gatekeeper' has been added now.

shelvenzhou commented on content/en-us/docs/khala-mining/1-1-hardware-requirements.md:
```diff
@@ -10,52 +10,301 @@ menu:
+- [x] A motherboard and BIOS that supports [Intel® SGX](https://www.intel.com/content/www/us/en/architecture-and-technology/software-guard-extensions.html) to run the [Trusted Execution Environment (TEE)](https://murdoch.is/talks/rhul14tee.pdf) is required.
+   - [Check BIOS compatability](#check-your-bios).
+- [x] A Solid-state drive (SSD) storage device
+   - Storing your blockchain data on a mechanical HDD will result in slow syncronosation speeds. At a minimum a 512GB SSD drive is recommended. 
```
```suggestion
   - Storing your blockchain data on a mechanical HDD will result in slow synchronization speeds. At a minimum a 512GB SSD drive is recommended. 
```

shelvenzhou commented on content/en-us/docs/khala-mining/1-1-hardware-requirements.md:
```diff
@@ -10,52 +10,301 @@ menu:
+      - Refer to [issue #554](https://github.com/Phala-Network/phala-blockchain/issues/554) for more info.
+- [x] Have a supported version of Ubuntu (18.04, 20.04, 21.04) installed and booted from it.
+   - See [Suported OS requirements](#supported-operating-systems). 
+- [x] You require a minimum amount of 3 PHA to mine
```
This sentence can be confusing, cuz miners may think 3 PHA is enough for mining.

hauni97 replied:
removed sentence & restructured

shelvenzhou commented on content/en-us/docs/khala-mining/1-1-hardware-requirements.md:
```diff
@@ -10,52 +10,301 @@ menu:
+
+> A mainboard supporting Intel® SGX and the BIOS settings listed below is required.
+
+### 1. Boot into BIOS. 
```
```suggestion
### 1. Boot into BIOS
```

shelvenzhou commented on content/en-us/docs/maintain/collator copy.md:
```diff
@@ -0,0 +1,136 @@
+---
```
better rename this file and remove the `copy` suffix

hauni97 replied:
@shelvenzhou yes, this file should be renamed to 'gatekeeper'.

hauni97 replied:
renamed

hauni97 reviewed (commented):
TODO: adjust file names & hide archived.

hauni97 commented:
# Fully redesigned mining guide and overall upgrades.

Improvements include collapsing sections (mining guide & developer guide), a wholly redesigned mining guide, and updated content throughout the wiki.
* Updating content from the current wiki
* Adding current wiki Chinese & French-language content (currently hidden)
* Cleaned up and organized the images folders under: /static/images for more straightforward future community contribution

hauni97 commented:
> 

@h4x3rotab & @shelvenzhou it is ready. Requires final approval.
